src/data/markdown/translated-guides/en/02 Using k6/06 Test life cycle.md

```diff
@@ -26,10 +31,26 @@ export function teardown(data) {
 
 </CodeGroup>
 
+## A quick overview of test stages
+
+This table provides the essential information about each stage.
+The rest of this page goes into deeper technical detail.
+
+| Test stage      | Used to                                                    | Example                                                                                 | Called                                                                             | Required? |
+|-----------------|------------------------------------------------------------|-----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|-----------|
+| **1. init**     | Load local files, import modules, declare global variables | Open JSON file, Import module                                                           | Once per VU\*                                                                      | Optional  |
+| **2. Setup**    | Set up data for processing, share data among VUs           | Call API to start test environment                                                      | Once                                                                               | Optional  |
+| **3. VU code**  | Run the test script                                        | Make https requests, validate responses                                                 | Per VU, for as many iterations and as long a duration as the test options required | Required       |
+| **4. Teardown** | Process result of setup code, stop test environment        | Validate that setup had a certain result, send webhook notifying that test has finished | Once per script                                                                    | Optional        |
+
+\* In cloud scripts, init code might be called more often.
+
+
+
 ## Init and VU stages
 
-Scripts must contain, at the very least, a `default` function - this defines the entry point
-for your VUs, similar to the `main()` function in many other languages:
+Scripts must contain, at the very least, a `default` function.
```

Comment:
Technically not - through the use of scenarios' `exec` setting you can define another exported function as the entry point for that scenarios. If all scenarios do that - k6 will not need a `default` function.